codegen/src/main/java/org/web3j/codegen/unit/gen/CompilerClassLoader.java

```diff
@@ -39,14 +39,14 @@
      * @param outputDir Directory where classes will be compiled.
      * @param urls Classpath URLs to compile the Java sources.
      */
-    CompilerClassLoader(final File outputDir, final URL... urls) {
+    public CompilerClassLoader(final File outputDir, final URL... urls) {
         super(CompilerClassLoader.class.getClassLoader());
         this.outputDir = outputDir;
         this.urls = urls;
     }
 
     @Override
-    protected Class<?> findClass(final String name) throws ClassNotFoundException {
+    public Class<?> findClass(final String name) throws ClassNotFoundException {
```

Comment:
We shouldn't change the visibility of a standard Java class method, it has been declared `protected` for some reason.

The method you want to use is `loadClass(String)`.